enterprise/templates/enterprise/emails/user_notification.html

```diff
@@ -0,0 +1,10 @@
+<p>{% if user_name %}Dear {{ user_name }},{% else %}Hi!{% endif %}</p>
+<p>
+You have been enrolled in <a href="{{ course_url}}">{{ course_name }}</a>, a {% if program_type %}{{ program_type }} {% endif %}{{ course_type }} offered by {{ organization_name }}. This {{ course_type }} begins {{ start_date | date:"F j, Y" }}. For more information, see <a href="{{ course_url}}">{{ course_name }}</a>.
+</p>
+<p>
+Thanks,
+</p>
+<p>
+The {{course_name}} team
+</p>
```

Comment:
An HTML email template should have a full `<html>` document - see http://localhost:8000/admin/bulk_email/courseemailtemplate/ on your devstack for an example. It would even be good to use that as a template here, since it's more detailed and pretty. (Arguably too detailed).

Also, admin users should have some way of seeing a default template - I think that's done somehow for http://localhost:8000/admin/bulk_email/courseemailtemplate/ but for this new template, I have no examples to work from in the django admin.